robottelo/datafactory.py

```diff
@@ -277,17 +317,14 @@ def valid_org_names_list():
 
 @datacheck
 def valid_usernames_list():
-    """Returns a list of valid user names.
-
-    Note: utf8 data is being rejected by the satellite server so it is not
-    included
-
-    """
+    """Returns a list of valid user names."""
     return [
         gen_string("latin1"),
         gen_string("alpha"),
         gen_string("alphanumeric"),
         gen_string("numeric"),
+        gen_string("utf8"),
+        gen_string('cjk'),
```

Comment:
We need to take care on length here since utf8 characters are multi-byte and AFAIK the validation counts the number of bytes an not characters.